.github/workflows/build-mudlet.yml

```diff
@@ -435,3 +435,29 @@ jobs:
           echo "CMakeOutput.log: "
           cat "${{runner.workspace}}/b/ninja/CMakeFiles/CMakeOutput.log"
         }
+
+    - name: Create PR to update development
+      if: github.event_name == 'push' && (github.event_name == 'developement' || startsWith(github.ref, 'refs/tags/Mudlet-'))
```

Comment:
Small typo in 'developement'

Reply:
The typo is still here.

Reflecting on this, this is not related to building Mudlet in itself. I think a separate workflow that only runs when we create a new release would be better.

Thanks for iterating on this!